packages/datadog-instrumentations/src/mongodb-core.js

```diff
@@ -58,22 +57,6 @@ addHook({ name: 'mongodb-core', versions: ['~3.1.10'], file: 'lib/wireprotocol/2
   return WireProtocol
 })
 
-addHook({ name: 'mongodb', versions: ['>=3.5.4 <4.11.0'], file: 'lib/utils.js' }, util => {
```

Comment:
It seems weird that this hook was removed but not replaced with anything. Do you know why it is no longer needed?